eZ/Publish/Core/Repository/URLService.php

```diff
@@ -256,7 +260,7 @@ protected function isUnique($id, $url)
         }
     }
 
-    private function createDateTime($timestamp)
+    private function createDateTime($timestamp): ?DateTimeInterface
```

Comment:
```suggestion
    private function createDateTime(int $timestamp): ?DateTimeInterface
```

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2854/commits/1719f1cf4a628cce40bbf098c2172c7ceae4d1f9